lib/instrumental/agent.rb

```diff
@@ -42,18 +44,28 @@ def initialize(api_key, options = {})
 
       # defaults
       # host:        collector.instrumentalapp.com
-      # port:        8000
+      # port:        8001
       # enabled:     true
       # synchronous: false
+      # secure:      true
+      # verify:      true
       @api_key         = api_key
       @host, @port     = options[:collector].to_s.split(':')
       @host          ||= 'collector.instrumentalapp.com'
-      @port            = (@port || 8000).to_i
+      desired_secure   = options[:secure].nil? ? allows_secure? : !!options[:secure]
+      if !allows_secure? && desired_secure
+        @logger.warn "Cannot connect to Instrumental via encrypted transport, SSL not available"
+        desired_secure = false
```

Comment:
Totally agree. I think our practice has been "never interrupt customer software in production" which I think we should hold to here, but I can absolutely see the case in development mode for these errors to be loud and cause fast failing.